src/lib.js

```diff
@@ -112,20 +112,22 @@ function deleteDir(ftp, dir) {
             console.log("Deleting directory:", dir);
             let dirNames =
                 lst
-                    .filter(f => f.type == 'd')
+                    .filter(f => f.type == 'd' && f.name != '..' && f.name != '.' )
```

Comment:
this makes sense - I'm wondering why I did not face any issues with my testing/use that would have led me to implement something like this already?

Reply:
Yes, maybe this depends on the ftp server for mine this came up for sure.